.gitignore

```diff
@@ -15,10 +15,9 @@ build/*
 *.perspective
 *.perspectivev3
 *.pbxuser
-*.xcworkspace
```

Comment:
The workspace now integrates the framework and the example app. So we should include it.